server/store/in_memory_store.go

```diff
@@ -28,6 +30,9 @@ func (store *InMemoryStore) GetKurtosisPackages(_ context.Context) ([]*generated
 	for _, kurtosisPackage := range store.packages {
 		packages = append(packages, kurtosisPackage)
 	}
+	sort.SliceStable(packages, func(i, j int) bool {
+		return strings.ToLower(packages[i].GetUrl()) < strings.ToLower(packages[j].GetUrl())
```

Comment:
I noticed youre sorting by alphabetical order in the get request here by URL and also sorting by alphabetical order in frontend side. Is this to ensure package catalog is in alphabetical order?

Reply:
yes or more generally deterministically sorted. We can sort by any key we want. The API should return results sorted so I added it there first. This is particularly important if you are paging results. I also added it here because I still want the data sorted even if the API changes sorting key.